### PR TITLE
Fix server host trimming in case server host is an object

### PR DESCRIPTION
**Pull request by nullivex**
This will fix a crash when the server value is returned as an object.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+                server: match[3].trim()
+              });
+              _this.lookup(addr, options, function(err, parts) {
+                if (err != null) {
+                  return done(err);
+                }
+                if (options.verbose) {
+                  return done(null, [
+                    {
+                      server: server.trim(),
```
Comment (NickLojas) [inline index.js]:
```suggestion
                      server: ('object' === typeof server) ? server.host.trim() : server.trim(),
```

**Comment by nullivex** [comment]
I agree with your additional change to the request. I applied that and it is ready for your review. @NickLojas thank you!

**Comment by hjr265** [comment]
@nullivex It seems the *.js was removed from .gitignore and that caused the compiled index.js file to be included. Can you please correct that?

**Comment by nullivex** [comment]
@hjr265 @NickLojas Oops, that file was not meant to be comitted a revised .gitignore and removed the index.js. How does this look?

Thank you in advance!

**Comment by hjr265** [comment]
@nullivex Looks good! Thank you

**Comment by hjr265** [comment]
@nullivex Wait... There is just one more irrelevant change: package.json.

Can you please correct that as well?

**Comment by hjr265** [comment]
@nullivex Thank you!

**Pull request merged**